uat/features/context_list_active.feature

```diff
@@ -1,9 +1,10 @@
+@contrib
 Feature: Context List Active
   In order to ensure that upgrades do not break existing functionality
   As an administrative user
   I want to ensure the Context List Active module is working properly
 
-  @api @javascript
+  @api @javascript @deploy @safe
```

Comment:
I'm on the fence on this one. context_list_active is not enabled by default in the `stanford` profile, and this feature enables it. I am leaning towards only marking things as `@deploy` and `@safe` in this uat directory if the `stanford` profile enables them by default. Maybe this directory should have its own README too.